### PR TITLE
Remove references of Stack Builder from EPRS docs

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

https://enterprisedb.atlassian.net/browse/DOCS-669 

References for Stack Builder Plus can remain.

### CODE REVIEW

**Comment by gvasquezvargas** [comment]
Hey @dwicinas! It seems like none of the edits I made affect the installation templates, but tagging you here for a double-check.

File: product_docs/docs/eprs/7/installing/windows.mdx [hunk 1]
```diff
@@ -33,19 +33,9 @@ In some versions of Windows, to invoke the installer with administrator privileg
 
 Proceed to [Using the graphical installer](#using-the-graphical-installer).
 
-## Using Stack Builder or StackBuilder Plus
+## Using StackBuilder Plus
 
-If you're using PostgreSQL, you can invoke the graphical installer with Stack Builder. See [Using Stack Builder](https://www.enterprisedb.com/docs/supported-open-source/postgresql/installing/03_using_stackbuilder/).
```
Comment (gvasquezvargas) [inline product_docs/docs/eprs/7/installing/windows.mdx:38]: @anwar-sahib 
How can customers on PostgreSQL install EPRS on Windows if Stack Builder is no longer available? 
Can they use Stack Builder Plus? Or do they have to go for the "direct installation" path (installation wizard)?

**Review by dwicinas — APPROVED** [review]
Looks good to me

File: product_docs/docs/eprs/7/installing/windows.mdx (hunk 1 above)
Comment (anwar-sahib) [reply product_docs/docs/eprs/7/installing/windows.mdx:38]: There is a way to directly install EPRS for Windows via website - https://www.enterprisedb.com/software-downloads-postgres#replication-server

**Review by anwar-sahib — APPROVED** [review]
Changes look fine.

**Pull request merged**